unittests/PhasarLLVM/DataFlowSolver/IfdsIde/Problems/CMakeLists.txt

```diff
@@ -24,7 +24,20 @@ else()
     IDEGeneralizedLCATest.cpp
 	IDEExtendedTaintAnalysisTest.cpp
 	IDETSAnalysisFileIOTest.cpp
+	IDELinearConstantAnalysisSwiftTest.cpp
   )
+  else()
+	set(IfdsIdeProblemSources
+	IFDSConstAnalysisTest.cpp
+	IFDSTaintAnalysisTest.cpp
+	IDEInstInteractionAnalysisTest.cpp
+	IDELinearConstantAnalysisTest.cpp
+	IDELinearConstantAnalysis_DotTest.cpp
+	IFDSUninitializedVariablesTest.cpp
+	IDEGeneralizedLCATest.cpp
+	IDEExtendedTaintAnalysisTest.cpp
+	IDETSAnalysisFileIOTest.cpp
+	)
```

Comment:
Fabian suggests another variable like IfdsIdeSwiftProblemSources that would be filled with the swift tests optionally. This if/else is a bit error prone and redundant.

Reply:
Makes sense. If you are fine with it we could already merge it. I'll add many Swift test cases later this week and during that process I have to edit this file anyway and will include this feedback then.